blazesym-go/README.md

```diff
@@ -0,0 +1,52 @@
+# blazsym-go
```

Comment:
I am fine with the name, but can we use the `go/` directory instead of `blazesym-go`? I'd rather not duplicate the name and keep it short.

Reply:
I thought there were some rules about matching paths and package names, but apparently it just works these days.

I moved it to just `/go` and it imports like this now:

```
blazesym "github.com/libbpf/blazesym/go"
```